waldo/data/paths.go

```diff
@@ -0,0 +1,32 @@
+package data
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/waldoapp/waldo-go-cli/lib"
+)
+
+func FindRepoSpecificPath() (string, error) {
+	path, err := lib.FindGitRepositoryPath()
+
+	if err == nil {
+		path = filepath.Join(filepath.Dir(path), ".waldo")
```

Comment:
This is why this pattern is bad I think. You meant if `err != nil` right ?

Reply:
Nope. In this case `err == nil` is exactly correct. It is changing the path from `/path/to/.git` to `/path/to/.waldo`, but only if `/path/to/.git` was successfully found.

Reply:
Pfiou sorry ! See I definitely did not read that correctly. I thought you had implemented a default if the path was not found